src/dart_dll.cpp

```diff
@@ -146,17 +146,22 @@ bool DartDll_Initialize(const DartDllConfig& config) {
 }
 
 Dart_Isolate DartDll_LoadScript(const char* script_uri,
-                                const char* package_config) {
+                                const char* package_config,
+								void* callbackPtr) {
   Dart_IsolateFlags isolate_flags;
   Dart_IsolateFlagsInitialize(&isolate_flags);
 
   char* error = nullptr;
   Dart_Isolate isolate = CreateIsolate(true, script_uri, "main", package_config,
-                                       &isolate_flags, nullptr, &error);
+                                       &isolate_flags, callbackPtr, &error);
 
   return isolate;
 }
 
+void* DartDll_GetCallbackData(void* isolate_group_data) {
```

Comment:
Any chance this could take a `Dart_Isolate` instead?

Reply:
I dont think so as there are api's to get the ptr that dont have an isolate like `Dart_GetNativeIsolateGroupData`